src/main/java/org/mule/extension/api/annotation/metadata/Content.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.extension.api.annotation.metadata;
+
+import static java.lang.annotation.ElementType.PARAMETER;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+import org.mule.api.metadata.resolving.MetadataContentResolver;
+import org.mule.extension.api.annotation.metadata.MetadataScope;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+/**
+ * Marks the parameter as the main input of operation and the one that will receive the the dynamic metadata
```

Comment:
Marks the parameter -> explain that is to be applied on arguments of a method from which an operation is derived.